bucket/sdcc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "http://sdcc.sourceforge.net",
+    "description": "A retargettable, optimizing Standard C (ANSI C89, ISO C99, ISO C11) compiler suite that targets the Intel MCS51 based, Maxim (Dallas) DS80C390 variants, Freescale (Motorola) HC08 based, Zilog Z80 based MCUs, Padauk and STMicroelectronics STM8.",
+    "version": "4.1.0",
+    "license": "GPL-2.0-only|GPL-3.0-only",
```

Comment:
i'm not sure what licenses sdcc uses.

Reply:
From the manual, it is a collection of tools under multiple licenses: 
* GPL2 
* LGPL2 
* GPL3 
* xlib/libpng 
* boost / BSL-1